### PR TITLE
Add zh_cn Language

### PR DESCRIPTION
**Pull request by MagmaBlock**
Took me more than an hour.

Hope Sodium gets better and better

### CODE REVIEW

**Comment by ArnaudLier** [comment]
There's #1008 already.

**Comment by Demonese** [comment]
感谢你的翻译，不过我已经在 #1008 提过添加中文本地化，目前都还没有合并

**Comment by MagmaBlock** [comment]
> 感谢你的翻译，不过我已经在 #1008 提过添加中文本地化，目前都还没有合并

汉化之前没有看GitHub，发拉取请求的时候也没有看    发重了ww

File: src/main/resources/assets/sodium/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  "sodium.options.use_fog_occlusion.name": "不要渲染蒙雾的区块",
+  "sodium.options.use_fog_occlusion.tooltip": "如果启用，完全被雾气效果隐藏的区块将不会被渲染，有助于提高性能。当雾的效果较重时（比如在水下时），改进会更明显。但在某些情况下，它可能会在天空和雾之间造成不理想的可视幻影。",
+  "sodium.options.use_entity_culling.name": "不要渲染不可视实体",
+  "sodium.options.use_entity_culling.tooltip": "如果启用，在渲染过程中，确定的不可见区块中的实体将被跳过。这可以通过避免渲染位于地下或墙壁后面的实体来帮助提高性能。",
+  "sodium.options.use_particle_culling.name": "不要渲染不可见粒子",
+  "sodium.options.use_particle_culling.tooltip": "如果启用，只有被确定为可见的粒子才会被渲染。当附近有许多粒子时，这可以大大改善帧率。",
+  "sodium.options.animate_only_visible_textures.name": "不要渲染不可见动画",
+  "sodium.options.animate_only_visible_textures.tooltip": "如果启用，只有被确定为可见的动态材质才会被更新。这在某些硬件上可以大大提升帧率，特别是在使用较大型的资源包时。如果你遇到一些动态材质始终是静态的问题，可以尝试禁用这个选项。",
+  "sodium.options.cpu_render_ahead_limit.name": "CPU 超前 GPU 帧数限制",
+  "sodium.options.cpu_render_ahead_limit.tooltip": "指定 CPU 在 GPU 完成渲染前可以等待几帧。很低或很高的数值可能会造成帧率不稳定。\nSpecifies the maximum number of frames the CPU can be waiting on the GPU to finish rendering. Very low or high values may create frame rate instability.",
```
Comment (liach) [inline src/main/resources/assets/sodium/lang/zh_cn.json:46]:
```suggestion
  "sodium.options.cpu_render_ahead_limit.tooltip": "指定 CPU 在 GPU 完成渲染前最多可以等待几帧。很低或很高的数值可能会造成帧率不稳定。",
```

**Comment by Pectacular** [comment]
> Thank you for the PR! **Unfortunately, we will not be accepting translation PR's directly, as we cannot verify them ourselves.** That being said, I do appreciate your effort!
> 
> **I am currently trying to determine/suggest an alternative to OneSky for translation hosting/crowdsourcing. Please see [this thread](https://discord.com/channels/602796788608401408/752637878462578778/922250280748335114) in the `#mod-discussion` channel of the CaffeineMC discord for more information.**
> 
> You don't have to—but I highly recommend—that you join [the official discord](http://caffeinemc.net/discord). This will give you access to important announcements about Sodium and translations, and allows you to give feedback/coordinate with other translators.
> 
> Thank you!


I don't think they accept translation PRs directly.

**Pull request closed without merging**